pages/api_differences.md.erb

```diff
@@ -0,0 +1,46 @@
+# API Differences Between REST and GraphQL
+
+Buildkite supports the use of both [REST](https://buildkite.com/docs/apis/rest-api) and [GraphQL](/docs/apis/graphql-api), but there is no full parity in the API between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+If you are only deciding which approach to take, you could consider GraphQL as your default answer to "how do I query a possibly-complicatedly-constrained subset of data?", and REST should be most helpful in a scenario of creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Exclusive REST features
+
+* Buildkite's terraform provider uses GraphQL where it can, but it falls back to using REST for setting [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
+* REST API can [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [`buildkite-agent annotate` command](/docs/agent/v3/cli-pipeline) from within a job).
```

Comment:
```suggestion
*  [Retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [`buildkite-agent annotate` command](/docs/agent/v3/cli-pipeline) from within a job).
```

I'm not clear if this is an only REST feature, or if the important information here is that you can't create Annoitations